pkg/resource/cluster.go

```diff
@@ -347,7 +347,12 @@ func (cluster Cluster) IsLoggingAPIEnabled() bool {
 	if cluster.Spec().CockroachDBVersion != "" {
 		version = cluster.Spec().CockroachDBVersion
 	} else if cluster.Spec().Image != nil && cluster.Spec().Image.Name != "" {
-		version = strings.Split(cluster.Spec().Image.Name, ":")[1]
+		split_result := strings.Split(cluster.Spec().Image.Name, ":")
+		if len(split_result) > 1 {
+			version = split_result[1]
+		} else {
+			return false
```

Comment:
Should we add a log line here or something? I'm just thinking that if this condition is hit (no semi-colon in the image name), how would the end user know?

Reply:
We have tried to add the log line before, but it seems that a parameter that defines the logger needs to be passed to this function. However, this parameter is currently not available in this function.

Reply:
Okay, I have added it in the latest commit.